modules.json

```diff
@@ -108,15 +108,6 @@
             "gatk4/variantrecalibrator": {
                 "git_sha": "e745e167c1020928ef20ea1397b6b4d230681b4d"
             },
-            "manta/germline": {
```

Comment:
why do the manta modules get moved?

Reply:
waiting for https://github.com/nf-core/modules/pull/1352